### DISCUSSION TITLE
Bot fully dies when some commands crash the bot

### DISCUSSION THREAD

**Discussion by rayanfer32** [phaticusthiccy/WhatsAsenaDuplicated#126] [Bug Discussion] [answered]
It seems that Plugins aren't well isolated from the main bot code, the bot doesn't respond sometimes even when it shows running in heroku logs.

**Comment by phaticusthiccy** [comment, marked as answer]
Each WhatsAsena module works in communication with each other. But one mistake does not create a chain reaction. Since it is modular, later add-ons may break functionality. 
At such times, report the issue to our WhatApp Help Group. You can also forward it to our [telegram](t.me/AsenaSupport) help group.

**Comment by rayanfer32** [comment]
Oh great.. ill send it right away next time.

**Comment by AFELI65** [comment]
> seems

H
